src/drivers/display/ft81x/lv_ft81x.c

```diff
@@ -0,0 +1,435 @@
+
+    if(lv_area_get_width(area) == hor_res) {
+        lv_ft81x_encode_write_address(encoded_address, address);
+        drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_ASSERT, NULL, 0);
+        drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, encoded_address, sizeof(encoded_address));
+        drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, px_map, lv_area_get_size(area) * 2);
+        drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_DEASSERT, NULL, 0);
+    }
+    else {
+        uint32_t area_height = lv_area_get_height(area);
+        uint32_t area_row_bytes = lv_area_get_width(area) * 2;
+        for(uint32_t i = 0; i < area_height; i++) {
+            lv_ft81x_encode_write_address(encoded_address, address);
+            drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_ASSERT, NULL, 0);
+            drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, encoded_address, sizeof(encoded_address));
+            drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, px_map, area_row_bytes);
+            drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_DEASSERT, NULL, 0);
+            address += disp_row_bytes;
+            px_map += area_row_bytes;
+        }
+    }
+
+    lv_display_flush_ready(disp);
+}
+
+static void delete_cb(lv_event_t * e)
+{
+    lv_display_t * disp = lv_event_get_target(e);
+    lv_ft81x_driver_data_t * drv = lv_display_get_driver_data(disp);
+    lv_free(drv);
+}
+
+static void lv_ft81x_cmd(lv_display_t * disp, uint8_t command, uint8_t parameter)
+{
+    lv_ft81x_driver_data_t * drv = lv_display_get_driver_data(disp);
+    uint8_t data[3] = {command, parameter, 0x00};
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_ASSERT, NULL, 0);
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, data, sizeof(data));
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_DEASSERT, NULL, 0);
+}
+
+static uint8_t lv_ft81x_read_8(lv_display_t * disp, uint32_t address)
+{
+    lv_ft81x_driver_data_t * drv = lv_display_get_driver_data(disp);
+    lv_ft81x_encode_read_address(&address, address);
+    uint8_t val;
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_ASSERT, NULL, 0);
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, &address, sizeof(address));
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_RECEIVE, &val, sizeof(val));
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_DEASSERT, NULL, 0);
+    return val;
+}
+
+static uint16_t lv_ft81x_read_16(lv_display_t * disp, uint32_t address)
+{
+    lv_ft81x_driver_data_t * drv = lv_display_get_driver_data(disp);
+    lv_ft81x_encode_read_address(&address, address);
+    uint16_t val;
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_ASSERT, NULL, 0);
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_SEND, &address, sizeof(address));
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_RECEIVE, &val, sizeof(val));
+    drv->spi_cb(disp, LV_FT81X_SPI_OPERATION_CS_DEASSERT, NULL, 0);
+    val = LE_TO_OR_FROM_NATIVE_16(val);
+    return val;
+}
+
+/*  static uint32_t lv_ft81x_read_32(lv_display_t * disp, uint32_t address)
```

Comment:
Will this function be used when this PR gets merged? If not I suggest to remove it